src/ecdsa/ssh.py

```diff
@@ -0,0 +1,80 @@
+import base64
+import binascii
+
+_SSH_ED25519 = b"ssh-ed25519"
+_SK_MAGIC = b"openssh-key-v1\0"
+_SK_START = b"-----BEGIN OPENSSH PRIVATE KEY-----\n"
+_SK_END = b"-----END OPENSSH PRIVATE KEY-----\n"
+_NONE = b"none"
+
+def _get_key_type(name):
+    if name == "Ed25519":
+        return _SSH_ED25519
+    else:
+        raise ValueError("Unsupported key type")
+
+class _Serializer:
+    def __init__(self):
+        self.bytes = b""
+
+    def put_raw(self, val):
+        self.bytes += val
+
+    def put_u32(self, val):
+        self.bytes += val.to_bytes(length=4, byteorder="big")
```

Comment:
I don't think this will be portable to old python versions...

Reply:
Ok, I'll take a look at this.

Reply:
Actually, what do you mean with old Python versions? Are you referring to end of life ones?

https://devguide.python.org/versions/

Reply:
Yes, we explicitly support python 2.6. Just because upstream doesn't support it, doesn't mean there aren't supported distributions of it.